homeassistant/components/switch/sony_projector.py

```diff
@@ -0,0 +1,102 @@
+"""
+Use SDCP network control to power on/off and check state of Sony projector.
```

Comment:
Actually - one last thing, we've changed the headers on files now:

This should read """Support for Sony projectors via SDCP network control.""" (with nothing else in the header).